config/locales/en.yml

```diff
@@ -641,6 +641,7 @@ en:
         case_type:
           income_tax_html: "<strong>Income Tax</strong><br>including tax paid through
```

Comment:
Oh wait - ignore me. I hadn't noticed that the text above has both <strong> and normal parts.